src/NServiceBus.AcceptanceTests/Performance/TimeToBeReceived/When_TimeToBeReceived_used_with_unobtrusive_mode.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace NServiceBus.AcceptanceTests.Performance.TimeToBeReceived
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_TimeToBeReceived_used_with_unobtrusive_mode : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Message_should_not_be_received()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Sender>()
+                .WithEndpoint<Receiver>()
+                .Run(TimeSpan.FromSeconds(10));
+
+            Assert.IsFalse(context.WasCalled);
+        }
+
+        public class Context : ScenarioContext
+        {
+            public bool WasCalled { get; set; }
+        }
+
+        class SendMessageWhileStarting : Feature
+        {
+            protected override void Setup(FeatureConfigurationContext context)
+            {
+                context.RegisterStartupTask(b => new SendMessageWhileStartingTask());
+            }
+        }
+
+        class SendMessageWhileStartingTask : FeatureStartupTask
+        {
+            protected override Task OnStart(IMessageSession session)
+            {
+                return session.Send(new MyCommand());
```

Comment:
we're using a startup task here because a regular when is only executed after both endpoint and receiver are started, right?

In other ttbr tests, we only check the delivery constraints I thought and I'm not sure whether we should move this test some something similar to make it less brittle (e.g. assume ttbr to work correctly when the delivery constraint is set)?